### PR TITLE
Update to add netfx-isolated to the feed.

### PR DESCRIPTION
**Pull request by fabiocav**

### CODE REVIEW

File: cli-feed-v4.json [hunk 1]
```diff
@@ -6119,6 +6119,199 @@
...
+            "sdk": {
+              "name": "Microsoft.Azure.Functions.Worker.Sdk",
+              "version": "1.3.0"
+            },
+            "default": false,
+            "toolingSuffix": "netfx-isolated",
+            "localEntryPoint": "dotnet.exe",
+            "targetFramework": "net472",
+            "itemTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.Functions.Worker.ItemTemplates/4.0.2057",
+            "projectTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.Functions.Worker.ProjectTemplates/4.0.2057",
```
Comment (soninaren) [inline cli-feed-v4.json:6245]: We would need a different set of packages for this

File: cli-feed-v4.json [hunk 2]
```diff
@@ -6119,6 +6119,199 @@
...
+          },
+          "netfx-isolated": {
+            "displayInfo": {
+              "displayName": ".NET Framework",
+              "hidden": false,
+              "displayVersion": "v4",
+              "targetFramework": ".NET Framework",
+              "description": "Isolated"
+            },
+            "capabilities": "isolated,net6",
```
Comment (soninaren) [inline cli-feed-v4.json:6235]: Should remove net6 from here

File: cli-feed-v4.json [hunk 3]
```diff
@@ -6119,6 +6119,199 @@
...
+              "netFrameworkVersion": "v6.0"
+            },
+            "linuxSiteConfig": {
+              "linuxFxVersion": "DOTNET-ISOLATED|6.0"
+            }
+          },
+          "netfx-isolated": {
+            "displayInfo": {
+              "displayName": ".NET Framework",
+              "hidden": false,
```
Comment (soninaren) [inline cli-feed-v4.json:6230]: Might want to set this to true in case we accidentally move pre-release to prod

File: cli-feed-v4.json [hunk 4]
```diff
@@ -6119,6 +6119,199 @@
...
+              "version": "1.3.0"
+            },
+            "default": false,
+            "toolingSuffix": "netfx-isolated",
+            "localEntryPoint": "dotnet.exe",
+            "targetFramework": "net472",
+            "itemTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.Functions.Worker.ItemTemplates/4.0.2057",
+            "projectTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.Functions.Worker.ProjectTemplates/4.0.2057",
+            "projectTemplateId": {
+              "csharp": "Microsoft.AzureFunctions.ProjectTemplate.CSharp.Isolated.3.x"
```
Comment (soninaren) [inline cli-feed-v4.json:6247]: Would probably need to update this on the template changes

File: cli-feed-v4.json [hunk 5]
```diff
@@ -6119,6 +6119,199 @@
...
+            "localContainerBaseImage": "DOCKER|mcr.microsoft.com/azure-functions/dotnet-isolated:4-dotnet-isolated6.0-appservice",
+            "serviceAppSettings": {
+              "FUNCTIONS_EXTENSION_VERSION": "~4",
+              "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated"
+            },
+            "windowsSiteConfig": {
+              "netFrameworkVersion": "v6.0"
+            },
+            "linuxSiteConfig": {
+              "linuxFxVersion": "DOTNET-ISOLATED|6.0"
```
Comment (soninaren) [inline cli-feed-v4.json:6258]: Not sure what should be the new value for this

File: cli-feed-v4.json [hunk 6]
```diff
@@ -6119,6 +6119,199 @@
...
+            "default": false,
+            "toolingSuffix": "netfx-isolated",
+            "localEntryPoint": "dotnet.exe",
+            "targetFramework": "net472",
+            "itemTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.Functions.Worker.ItemTemplates/4.0.2057",
+            "projectTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.Functions.Worker.ProjectTemplates/4.0.2057",
+            "projectTemplateId": {
+              "csharp": "Microsoft.AzureFunctions.ProjectTemplate.CSharp.Isolated.3.x"
+            },
+            "localContainerBaseImage": "DOCKER|mcr.microsoft.com/azure-functions/dotnet-isolated:4-dotnet-isolated6.0-appservice",
```
Comment (soninaren) [inline cli-feed-v4.json:6249]: also this one.

**Review by soninaren — CHANGES_REQUESTED** [review]

File: cli-feed-v4.json (hunk 2 above)
Comment (phil-allen-msft) [reply cli-feed-v4.json:6235]: Please leave net6 here; the inproc (func.exe/func.dll) is compiled with net6, as is the extensions project.

File: cli-feed-v4.json (hunk 5 above)
Comment (soninaren) [reply cli-feed-v4.json:6258]: You are right, this won't run on linux. Would deleting this section cause issues with VS?

File: cli-feed-v4.json (hunk 2 above)
Comment (phil-allen-msft) [reply cli-feed-v4.json:6235]: Actually, because of the 'publish to Linux' comments, likely this should also include "netfxisolated"

**Pull request closed without merging**